src/custom-elements/src/components/time/time.tsx

```diff
@@ -336,7 +371,7 @@ export class Time {
         let periodStatus = TIME_CLASSES.PERIOD;
 
         if (period === this._period) {
-          periodStatus += ' -active';
+          periodStatus += ` ${ACTIVE_CLASS}`;
```

Comment:
Is this correct?

Reply:
Yes, because periodStatus already includes `TIME_CLASSES.PERIOD`. This adds `-active` to the existing string.